### DISCUSSION TITLE
LF APAC 开源布道者分享系列之三：演讲这事很重要！ - 开源之道

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#26] [Announcements]
# LF APAC 开源布道者分享系列之三：演讲这事很重要！ - 开源之道

https://opensourceway.community/posts/lfapac-evangelist-sharing/why-we-need-speech-all-the-time/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

推荐汉语演讲核心力量神书: [中国传统相声大全（全四册） (1993)](https://book.douban.com/subject/1214873/)

要知道, 汉语是世界上最复杂的自然语言, 
而且中华文明从诞生尹始就没断绝过, 这其中积累的所有语言致高成果, 都在相声艺术中了,
也就是 传统段子, 
其实, 每个传统段子, 都是一个高度完备/可复用的汉语华彩范式,
可以套用在所有需要语言的场景中;

当然, 并不是要将每次演讲都变成 talk show ,
但是, 将语言这一工具, 掌握到自如, 可以根据现场反响, 自动调整到最佳互动氛围中,
那么, 无论什么类型的发言, 至少受听度就打满了;

其它的技巧, 其实也不多, 就两个:

- 多练习
- 要克制, 每次演讲, 可能将一件事儿说明白就很不得了了, 千万别唠叨太多内容, 通过语声, 很少有人能在42分钟以内记忆清楚 3+-2 件事儿...
